video-store/src/main/java/com/video/store/VideoStoreApplication.java

```diff
@@ -0,0 +1,96 @@
+package com.video.store;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+import org.json.simple.JSONObject;
+import org.json.simple.JSONArray;
+import org.json.simple.parser.JSONParser;
+
+import java.io.FileReader;
```

Comment:
Nitpick but most teams will require that you put these in alphabetical order. Your IDE might even have an option to right-click and alphabetize these for you.